frontend/src/components/user/ATSUserLinkModal.vue

```diff
@@ -66,9 +66,22 @@ async function searchATSUsers() {
   }
 }
 
-onMounted(async () => {
-  firstName.value = fName;
-  lastName.value = lName;
+onMounted(() => {
+  if (submission.contactFirstName && submission.contactLastName) {
+    firstName.value = submission.contactFirstName;
+    lastName.value = submission.contactLastName;
+  }
```

Comment:
Are these (any the other state values) needed if they all exist in the `proponent` state value?

Reply:
removed proponent state, it was leftover from the redesign, didn't catch it. thanks